debian/patches/no-nocloud-network.patch

```diff
@@ -16,11 +16,117 @@ Last-Update: 2024-08-02
              LOG.debug("Using seeded cache data from %s", seedfrom)
  
              # Values in the command line override those from the seed
-@@ -199,7 +199,6 @@ class DataSourceNoCloud(sources.DataSour
+@@ -199,7 +199,6 @@
              )
              mydata["user-data"] = ud
              mydata["vendor-data"] = vd
 -            mydata["network-config"] = network
```

Comment:
We could drop this from the patch to reduce line length if we wanted, since the default value is None and as the patch is written that's what gets returned.

Reply:
+1 thanks @holmanb will do